### PR TITLE
send resolved email and make user can config repeat interval

### PR DESCRIPTION
**Pull request by xudifsd**
@squirrelsc think there are too many same alert email, so maybe should enlarge the interval of same alerts, I make it as configurable and default to our old value.

At the same time, it would be useful to send resolved email to notify problem solved.

### CODE REVIEW

File: src/alert-manager/config/alert-manager.md [hunk 1]
```diff
@@ -50,4 +53,5 @@ alert-manager:
     host: master_ip
     url: "http://master_ip:9093"
     use-pylon: False
+    repeat-interval: 4h
```
Comment (squirrelsc) [inline src/alert-manager/config/alert-manager.md]: we can extend it to 24h, and depends on resolved mails.

File: src/alert-manager/deploy/alert-configmap.yaml.template [hunk 2]
```diff
@@ -18,12 +18,13 @@ data:
...
       group_interval: 5m
-      repeat_interval: 4h
+      repeat_interval: {{ cluster_cfg["alert-manager"]["repeat-interval"] }}
       group_by: [alertname]
     receivers:
     - name: "pai-alert"
       email_configs:
         - to: {{ alert_info["receiver"] }}
+          send_resolved: true
           html: '{{ "{{" }} template "email.pai.html" . {{ "}}" }}'
```
Comment (squirrelsc) [inline src/alert-manager/deploy/alert-configmap.yaml.template]: Let's make sure resolve mail have a green title.

File: src/alert-manager/config/alert-manager.md (hunk 1 above)
Comment (xudifsd) [reply src/alert-manager/config/alert-manager.md]: yes, I provide this as a config, user who want to extend can extend when deploying.

File: src/alert-manager/deploy/alert-configmap.yaml.template (hunk 2 above)
Comment (xudifsd) [reply src/alert-manager/deploy/alert-configmap.yaml.template]: I do not know how to pass this into template to generate a different email.

File: src/alert-manager/deploy/alert-templates/pai.tmpl [hunk 3]
```diff
@@ -43,7 +43,11 @@ SOFTWARE.
       <div style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 14px; max-width: 600px; display: block; margin: 0 auto; padding: 0;">
         <table width="100%" cellpadding="0" cellspacing="0" style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 14px; border-radius: 3px; background-color: #fff; margin: 0; border: 1px solid #e9e9e9;" bgcolor="#fff">
           <tr>
+            {{ if gt (len .Alerts.Firing) 0 }}
```
Comment (squirrelsc) [inline src/alert-manager/deploy/alert-templates/pai.tmpl]: please double check firing and resolving won't be in the same mail.

Comment (xudifsd) [reply src/alert-manager/deploy/alert-templates/pai.tmpl]: Yes, I've already checked.

**Pull request merged**